### ISSUE TITLE
Accessibility Feedback - Midpoint Review - BAH/Sitewide Content VRRAP static page

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#23374]
## VFS acceptance criteria
- [ ] Let VSP know when the **Must** feedback has been incorporated 
- [ ] Leave any comments for feedback you decide _not_ to take
- [ ] VFS team closes the ticket

## Thoughts/questions
-

## Feedback
- **Must** = the feedback must be applied
- **Should** = the feedback is best practice
- **Consider** = suggestions/enhancements

**Must:**

**Should:**

**Consider:**

## VSP directions
- Update "Issue Title"
- Link to collab cycle touchpoint meeting epic
- Add to the Feedback pipeline on the VSP Product Support board
- Add your feedback
- Add assignees based on collab cycle touchpoint
  - **Design Intent**: VFS designer, VFS PM (optional), yourself (optional)
  - **Usability Testing Prep**: VFS PM, yourself (optional)
  - **Staging Review**: VFS PM, yourself (optional)

**Comment by bethpottsVADEPO** [comment]
@RLHecht Can you please make the header in the blue box an H2? Also see Trevor's "should" feedback.

**Comment by RLHecht** [comment]
I've changed the header size in the blue box. To do this, you can't use the standard Q&A for featured content because it automatically makes the question an H3. Instead I made it a rich text field and turned the question into an H2. If these are supposed to always be H2s going forward the template within the CMS may need to be fixed. 

<img width="1068" alt="Screen Shot 2021-04-21 at 9 08 58 AM" src="https://user-images.githubusercontent.com/48728214/115558936-3777a180-a281-11eb-8313-e074cce0cc3e.png">


Until we publish the page, we can't display the full breadcrumbs but this will have Veteran Rapid Retraining Assistance in it.

**Comment by bethpottsVADEPO** [comment]
Thanks @RLHecht for changing that header. Would you discuss this accessibility request with the CMS team as far as the best approach going forward with the header in the blue box--should we create the blue box as rich text as you did or is there a different/better approach?

And would you reach out to the VA 508 office for a review of the high-demand jobs PDF? @1Copenut can you put that contact info here in this ticket for Randi?

**Comment by 1Copenut** [comment]
Sure thing! @RLHecht below are the steps to submit a document review request. LMK if you have questions or need additional info.

### Submitting a YourIT! request

* Access YourIT! from the VA network
* In the search box, enter "508 audit request" and click on the top result. Top result should be "508 Accessibility Audit Request".
* You'll get an input form to enter relevant data
* At the bottom of the form, select "Documents" from the Product Type dropdown
* Select "Documents" from the Product Type dropdown
* Upload the document(s) from your computer
* You should receive an email confirmation with a linked ticket number

**Comment by RLHecht** [comment]
Thanks, @1Copenut I just submitted the 508 audit request. Ticket number REQ4708690

**Comment by RLHecht** [comment]
@1Copenut here's the 508 audit of the PDF. Do we need to reach out to the EDU office to fix what needs to be fixed? FYSA @bethpottsVADEPO 

[30518 High-Demand Jobs PDF Interim Audit Report 05 12 2021.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/6467508/30518.High-Demand.Jobs.PDF.Interim.Audit.Report.05.12.2021.docx)

**Comment by bethpottsVADEPO** [comment]
It looks like there are 2 defects we need to make sure the EDU team addresses. @1Copenut Can you take a look at the 508 report here and let us know if this will do it? I.e, this PDF will be considered accessible once these issues are addressed?

@RLHecht I can reach out to the EDU contact for VRRAP and find out who to route the 508 required changes to...

**Comment by bethpottsVADEPO** [comment]
I sent the 508 report to Terry Warren in Education Service. I think he'll be able to make the updates.

**Comment by RLHecht** [comment]
@1Copenut Terry Warren, who is providing us with the PDF for the VRRAP page is having a hard time figuring out how to fix the issues that the 508 office came back with (the doc is in a previous comment here). Would you be able to provide some guidance I can pass along to him? He sent me the Excel sheet with the information he converted to a PDF but he said when he does that the rows are not standard and he fails the logical flow.

**Comment by bethpottsVADEPO** [comment]
@RLHecht Is this resolved now. Can we close this ticket?

**Comment by RLHecht** [comment]
@1Copenut Here's the PDF we just discussed that has the 508 issues: https://benefits.va.gov/GIBILL/docs/vrrap-high-demand-occupation-list.pdf

And here is the newer PDF we said was an issue with converting it from Excel:
https://benefits.va.gov/gibill/docs/VRRAP_ApprovedEducationalInstitutions.pdf

Thanks so much for your help with these!

**Comment by 1Copenut** [comment]
@RLHecht I took a look at the first PDF and think I understand the issues that were raised. The PDF itself passed the Acrobat accessibility scan, but looking at the tags, were were no headings, and the lists were marked up as tables. I'd recommend going back to the original Word document and setting the first heading as an H1, then styling it to match the current. Same with the sub-header as an H2, and lists as ordered lists.

Attaching a couple of screenshots. I'll comment on the second PDF in a second comment below.

---
![Screen Shot 2021-08-19 at 4 45 21 PM](https://user-images.githubusercontent.com/934879/130149328-9cd6e29e-9b02-4b31-b1b2-d210b2b29bff.png)

---
![Screen Shot 2021-08-19 at 4 45 00 PM](https://user-images.githubusercontent.com/934879/130149355-73b4506f-377f-4d10-ac9f-f3c471c18136.png)

**Comment by 1Copenut** [comment]
@RLHecht The second PDF that was built out from a spreadsheet would be better served as a spreadsheet. The PDF failed validation for table headers and regularity, meaning screen readers won't be able to assign the relationship between columns, the college row headers, and possibly page breaks.

Angela said a properly formatted spreadsheet is accessible to screen readers because it maintains the relationships from header rows, across pages, and users can skip around cells as they need.

The other (more work intensive) option would be to recreate the spreadsheet in Word, using the Word table maker and creating explicit header rows, and individual row heads for the colleges. The Word doc could be exported to PDF, then validated in Acrobat.

LMK if you'd like to discuss briefly about these options.

**Comment by RLHecht** [comment]
Thanks so much for looking into these @1Copenut So for this second PDF, your suggestion is to put it up on as an Excel file rather than a PDF? @bethpotts I believe we've done this before on other pages but not often. Just want to confirm this is OK before I go back to the stakeholder.

**Comment by 1Copenut** [comment]
@RLHecht Yes, for the second PDF it would be better to put it on the site as an Excel file, as long as the accessibility checks in Excel all pass. The complex relationship between table columns and row headers (college) with cities, certificates, states, etc. will be easier to keep consistent this way. Angela Fowler confirmed this in a discussion yesterday that a native spreadsheet is a better way forward.

I've had good luck [using the built-in accessibility checker](https://support.microsoft.com/en-us/office/improve-accessibility-with-the-accessibility-checker-a16f6de0-2f39-4a2b-8bd8-5ad801426c7f) with Microsoft desktop programs.

/cc @AngelaFowler82 for awareness
